docs/reference/api/docker_remote_api.md

```diff
@@ -64,12 +64,20 @@ without protocol. Throughout this structure, double quotes are required.
 
 ## Using Docker Machine with the API
 
-If you are using `docker-machine`, the Docker daemon is on a virtual host that uses an encrypted TCP socket. This means, for Docker Machine users, you need to add extra parameters to `curl` or `wget` when making test API requests, for example:
+If you are using `docker-machine`, the Docker daemon is on a virtual
+host that uses an encrypted TCP socket. This means, for Docker Machine
```

Comment:
Heck, we're in the `using docker machine` section, you don't even need anything before the first comma. Also, given all the flags are called `tls`-something, can we work that in there instead of `encrypted tcp`? I don't want to be rewriting everything, but let's be both consistent and specific if possible.